.github/workflows/contribution_evaluation.yaml

```
@@ -0,0 +1,227 @@
+          (cd src && mkdir build && cmake -S . -B build)
+          build-wrapper-linux-x86-64 --out-dir ${{ env.BUILD_WRAPPER_OUT_DIR }} cmake --build src/build/ --config Release
+          cat ${{ env.BUILD_WRAPPER_OUT_DIR }}/compile_commands.json
+
+      - name: SonarQube Scan
+        uses: SonarSource/sonarqube-scan-action@v4
+        env:
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+        with:
+          args: >
+            --define sonar.cfamily.compile-commands="${{ env.BUILD_WRAPPER_OUT_DIR }}/compile_commands.json"
+            --define sonar.coverage.exclusions=**/* 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)